codegen/src/lib.rs

```diff
@@ -1,10 +1,10 @@
-mod main;
+mod shuttle_main;
```

Comment:
Why was the rename needed? :smile:

Reply:
Woops, forgot to write a note about this. It's because clippy wouldn't allow it anymore, and I couldn't find any way to allow it, there was no `#[allow(special-module-name)]` or similar  that I could find. 

Ref failed CI: https://app.circleci.com/pipelines/github/shuttle-hq/shuttle/1176/workflows/31f0e441-7778-462c-a4ce-2aca2b5eb2ba/jobs/19417